components/meter/meter-mixin.js

```diff
@@ -21,6 +21,11 @@ export const MeterMixin = superclass => class extends LocalizeCoreElement(superc
 			 * @type {string}
 			 */
 			text: { type: String },
+			/**
+			 * Displays the text as a label beneath the meter
+			 * @type {boolean}
+			 */
+			textHidden: { type: Boolean, attribute: 'text-hidden' },
```

Comment:
It looks like `d2l-meter-circle` makes use of this attribute in the CSS, so we'd want to specify `reflect: true` here.